examples/hooks/schema.ts

```diff
@@ -106,9 +131,14 @@ export const lists: Lists = {
           return resolvedData;
         },
       },
-      validateInput: ({ context, inputData, addValidationError }) => {
+      validateInput: ({ context, operation, inputData, addValidationError }) => {
```

Comment:
My intent is to replace this soon with per-operation typed equivalent, equivalent to `validate.create`, `validate.update` and `validate.delete`, or similar.